ariadne/src/Ariadne/Wallet/Cardano/Kernel/Word31.hs

```diff
@@ -26,6 +26,10 @@ instance Exception Word31Exception where
     displayException (Word31Overflow n) =
         "Word31: " <> show n <> " is too large"
 
+instance Enum Word31 where
+    toEnum = unsafeMkWord31 . fromIntegral
```

Comment:
Are you sure `unsafeMkWord31` should be used here? Where is this instance needed anyway?

Reply:
I started checking it and ended up doing `stack clean --full` :unamused:

Reply:
Anyway, `unsafeMkWord31` is ok, because `toEnum` is partial anyway. It's better to add a comment about it, though.

Reply:
```
/home/gromak/Work/ariadne/ariadne/src/Ariadne/Wallet/Backend/KeyStorage.hs:298:9: error:
    • No instance for (Enum Word31)
        arising from a use of ‘findFirstUnique’

```